SemiRP/Commands/AdminCommands.cs

```diff
@@ -19,7 +19,7 @@ private static void Help(Player sender)
             if (!sender.AccountData.HavePerm("admin.cmds.help"))
                 return;
 
-            Chat.AdminChat(sender, "help, goto, gethere, slap, freeze, unfreeze, pm, perm, give, set");
+            Chat.AdminChat(sender, "help, goto, gethere, slap, freeze, unfreeze, pm, kick, perm, v, set, give, delete");
```

Comment:
The help is in complete re-writing, you don't need to add this here, it will be automatic.